version1.go

```diff
@@ -21,6 +21,8 @@ func NewUUID() (UUID, error) {
 	if nodeID == zeroID {
 		setNodeInterface("")
 	}
+	var nodeIDCopy [6]byte
```

Comment:
nodeIDCopy := nodeID

But I think better would be to take the block of code from lines 29-24 and put them at the end of the function and then move line 43 to be prior to the unlock.  This would avoid adding new lines of code and also using the extra variable.

Reply:
e80e2ec should fix it